sdks/python/apache_beam/runners/portability/portable_runner_test.py

```diff
@@ -241,7 +241,8 @@ def process(self, kv, index=beam.DoFn.StateParam(index_state_spec)):
           equal_to(expected))
 
   # Inherits all other tests from fn_api_runner_test.FnApiRunnerTest
-
+  def test_pardo_timers_clear(self):
```

Comment:
Doesn't this turn off the test for `FlinkRunnerTest` and anything else that extends `PortableRunner`?

Reply:
Ah yes. I thought `FlinkRunnerTest` is inherited from `FnRunnerTest`.

Reply:
This may also be used for Spark. Skip should probably only occur in leaf classes that strictly need it. Otherwise, we are almost back to what we had before ;-)